backend/db/db.go

```diff
@@ -36,7 +40,17 @@ func New(config *ini.File) (*DatabaseStruct, error) {
 
 	tableNames = append(tableNames, "DEPARTMENT")
 
-	db, err := sql.Open("mysql", fmt.Sprintf("%s:%s@/%s%s", username, password, endpoint, databaseName))
+	// TODEL: must delete, for back compatibility temporarily
+	tableNames = append(tableNames, "CLASS_DATA")
+
+	db, err := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s)/%s", username, password, endpoint, databaseName))
+	// db, err := sql.Open("mysql", fmt.Sprintf("%s:%s@/%s%s", username, password, endpoint, databaseName))
```

Comment:
I don't know how the original code works but the endpoint is not at the correct place